frontend/src/components/GMap/GMap.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useState} from 'react';
+import GoogleMapReact from 'google-map-react';
+import './GMap.css';
+import Geocode from "react-geocode";
+
+Geocode.enableDebug();
+Geocode.setRegion("us");
+
+interface MarkerProps {
+  lat: number,
+  lng: number,
+  text: string
+}
+
+const Marker: FC<MarkerProps> = (props: MarkerProps) => {
+  return (
+    // Style with classname doesn't work for some reason here
+    <div className="GMapMarker" style={{
+    position: 'absolute',
+  width: 20,
+  height: 20,
+  left: -10,
+  top: -10,
+
+  border: '5px solid #f44336',
+  borderRadius: 20,
+  backgroundColor: 'white',
+  textAlign: 'center',
+  color: '#3f51b5',
+  fontSize: 16,
+  fontWeight: 'bold',
+  padding: 4
+    }}>
+      {props.text}
+    </div>
+  );
+}
+
+interface GMapProps extends google.maps.MapOptions{
+  droneAddrs: string[],
+  onClick?: (e: google.maps.MapMouseEvent) => void,
+  onIdle?: (map: google.maps.Map) => void,
+}
+const MAPS_KEY = "AIzaSyCaO9NsoZHXDEfqqHU6w2o_51bFiLV4tVc"
+Geocode.setApiKey(MAPS_KEY)
+
+// const query = "https://maps.googleapis.com/maps/api/geocode/json?key=" + MAPS_KEY + "&address=";
+
+const GMap: FC<GMapProps> = (props: GMapProps) => {
+  const addrs = props.droneAddrs;
+  console.log(addrs)
+  const [latd1, setlatD1] = useState(40.4)
```

Comment:
maybe define these numbers somewhere so we know what they are

Reply:
OK